procwrap/procwrap.py

```diff
@@ -2,39 +2,157 @@
 
 import os
```

Comment:
That is a good idea.
Is there any way we can still keep the following command though ?
`procwrap --enable-nsq <command>` works.
@prashanthellina 
With basescript this would become `procwrap --enable-nsq run <command>`

And we will have to disable the periodic stats on the wrapper process once we implement metrics as part of basescript.